.github/ISSUE_TEMPLATE/feature-spec.md

```diff
@@ -14,8 +14,12 @@ _As <persona>, I can <what?> so that <why?>_
 _What does the ideal solution look like?_
 
 ## MVP
-_What will be built as an MVP? (Please refer to our [wiki](https://github.com/opencollective/opencollective/wiki/Feature-Spec-Template) for what's acceptable as an MVP.)_
-
+_What will be built as an MVP?_
+  
+<!-- 
+MVPs should be contained to a minimum number of users. The goal is to test & get feedback or unblock a power user/org. MVPs can’t be released system wide. They need to incorporate feedback and must be then iterated into v1. MVPs need to be cohesive with design and tested.
+-->
```

Comment:
to be operfectly honest I hadn't noticed that there was a wiki over there 🙃 

In my opinion an MVP can be part of a limited release or it can be a minimum iteration of a product released to everyone. I dont think the guidance should bind someone to a choice at this point so let's remove this.

Reply:
Removed the whole comment.